libs/checkpoint-sqlite/langgraph/checkpoint/sqlite/__init__.py

```diff
@@ -519,7 +519,7 @@ async def alist(
             Use list() instead, or consider using [AsyncSqliteSaver][langgraph.checkpoint.sqlite.aio.AsyncSqliteSaver].
         """
         raise NotImplementedError(_AIO_ERROR_MSG)
-        yield
```

Comment:
No this is here to ensure the method is a generator instead of a function, it cannot be removed